torch/utils/cpp_extension.py

```diff
@@ -828,6 +828,35 @@ def CUDAExtension(name, sources, *args, **kwargs):
                 cmdclass={
                     'build_ext': BuildExtension
                 })
+
+    Compute capabilities:
+
+    By default the extension will be compiled to run on all archs of the cards visible during the
+    building process of the extension, plus PTX. If down the road a new card is installed the
+    extension may need to be recompiled.
+
+    If a visible card has a compute capability (CC) that's newer than the newest version for which
+    your nvcc can build fully-compiled binaries, Pytorch will make nvcc fall back to building
+    kernels with the newest version of PTX your nvcc does support (see below for details on PTX).
+
+    However, you can explicitly specify which archs you want the extension to support like so:
+
+    TORCH_CUDA_ARCH_LIST="6.1 8.6" python build_my_extension.py
+    TORCH_CUDA_ARCH_LIST="5.2 6.0 6.1 7.0 7.5 8.0 8.6+PTX" python build_my_extension.py
+
+    The +PTX option causes extension kernel binaries to include PTX instructions for the specified
+    compute capability (CC). PTX is an intermediate representation that allows kernels to
```

Comment:
above changes define CC, now we can replace "compute capability (CC)" with just "CC"

Reply:
done